cuda_core/tests/test_module.py

```diff
@@ -100,6 +111,13 @@ def test_object_code_load_cubin(get_saxpy_kernel):
     mod.get_kernel("saxpy<double>")  # force loading
 
 
+def test_object_code_load_ptx(get_saxpy_kernel_ptx):
```

Comment:
This test looks fine. I am just hesitant about testing this. Ideally we want people to JIT-compile a PTX to CUBIN via `Program`, because otherwise JIT'ing happens at the module load time, and we purposely don't take any JIT options in the constructor so there is no way (unlike `Program`) to influence JIT behavior. It might be OK, though, but I'd like to know your thoughts @brandon-b-miller @ksimpson-work.

Reply:
I see your hesitation. I think its fair to say to the user that if they want to use this API, it is their responsibility to ensure the PTX is appropriately prepared for whatever downstream link they have planned, otherwise, things may go wrong.